frontend/i18n/nl/docusaurus-plugin-content-docs/current/client/ClientCommands.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Client commands"
+sidebar_label: "Client commands"
+description: Lijst van alle client commands.
+---
+
+## Commands
+
+| Command        | Beschrijving                                                                                                                                                                                                                                                                                                                     |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| /quit (/q)     | Deze command spreekt voor zich: het sluit het spel af. Je kunt ook /q gebruiken; dat is dezelfde command, maar korter. |
```

Comment:
Volgens mij is _command_ onzijdig, dus: _dit_ command_, _het_zelfde command, ...